MusicNotationKit/MusicNotationKit/Tuplet.swift

```diff
@@ -102,6 +102,14 @@ extension Tuplet: NoteCollection {
 	internal var noteCount: Int { return notes.count }
 }
 
+extension Tuplet: CustomDebugStringConvertible {
+	public var debugDescription: String {
+		let result:String = String(format:"tuplet(notes: \n %@)",
```

Comment:
I think you can leave the `notes:` out of it. I think it should be obvious that those are the notes contained in the tuplet. Plus, the less extraneous text the better.